specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/securitydomain.json

```diff
@@ -222,8 +222,14 @@
     },
     "TransferKey": {
       "properties": {
+        "key_format": {
+          "type": "string",
+          "default": "jwk",
+          "description": "Specifies the format of the transfer key"
```

Comment:
Should this be an enum, even if a value of only "jwk"? How do the customers know what to put here (especially as newer versions maybe support newer formats), or was this part of the opaque data that they shouldn't neeed to know about?